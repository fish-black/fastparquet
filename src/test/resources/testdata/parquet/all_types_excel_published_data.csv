133499892,"Alice  ",24,13000.44545898764543489,1.249434,"1984-04-08","2007-05-04 08:02:11","08:02:11","2000-04-01T00:00:00.000Z","24"
133499834,"  Bob",25,840,1.1785,"1987-04-08","2008-06-04 08:02:11.566","09:04:11.8","2000-04-01T00:00:00.000Z","1"
133493498,"John",34,14045.9,1.15,"1989-08-24","2010-07-12 11:02:11.8","09:34:45.56","2000-07-01T00:00:00.000Z","15"
"","",,,,"","","","",""
"","",,,,"","","","",""
"","",,,,"","","","",""
"","",,,,"","","","",""
"","",,,,"","","","",""
"","",,,,"","","","",""