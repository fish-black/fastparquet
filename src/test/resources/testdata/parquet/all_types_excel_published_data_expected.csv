133499892,Alice  ,24,13000.445458987645,1.249434,1984-04-08,2007-05-04 08:02:11.0,08:02:11,2000-04-01,24
133499834,  Bob,25,840,1.1785,1987-04-08,2008-06-04 08:02:11.566,09:04:11.800,2000-04-01,1
133493498,John,34,14045.9,1.15,1989-08-24,2010-07-12 11:02:11.800,09:34:45.560,2000-07-01,15
,,,,,,,,,
,,,,,,,,,
,,,,,,,,,
,,,,,,,,,
,,,,,,,,,
,,,,,,,,,
